src/MergeContext.js

```diff
@@ -138,6 +138,17 @@ class MergeContext {
         return tagCommit.tree.sha === prCommit.tree.sha;
     }
 
+    // whether the tag commit and the base HEAD are 'diverged'
```

Comment:
If possible, avoid describing foo() as "does foo". For the reader who does not know what "diverged" means in this context, the current description is not very useful.

s/tag commit/tagged commit/ or s/tag commit/tag/ or /tag commit/staged commit/

Reply:
Done.